contrib/kube-prometheus/docs/developing-alerts-and-dashboards.md

```diff
@@ -22,6 +24,16 @@ The `ConfigMap` that is generated and holds the dashboard definitions can be fou
 
 As Grafana's support for applying dashboards from files is limited a sidecar (called "grafana-watcher") was implemented. It watches the dashboard definitions provided through the `ConfigMap` and ensures that Grafana's SQLite database is in sync with the dashboard definitions.
 
-To edit/create a dashboard login to Grafana and modify and save the dashboard. Then download the dashboard definition in Grafana through `Share` -> `Export` -> `Save to file`. Move the file to `assets/grafana/` and re-generate the manifests.
+To edit/create a dashboard login to Grafana and modify and save the dashboard. Then download the dashboard definition in Grafana through `Share` -> `Export` -> `Save to file`. Move the file to `assets/grafana/` and re-generate the manifests (executing `make generate` from kube-prometheus base directory).
 
-Then the generated manifest can be applied against a Kubernetes cluster.
+Note: The dashboard json file to be copied in `assets/grafana/` should be suffixed with `-dashboard.json`, otherwise it won't be processed by `make generate`.
+
+Then the generated manifest can be applied against a Kubernetes cluster with something like:
+```
+kubectl -n monitoring apply -f manifests/grafana/grafana-dashboards.yaml
+```
+
+And to apply the changes, restart grafana:
+```
+kubectl -n monitoring delete pod -l app=grafana
```

Comment:
This should not be necessary. The grafana-watcher should pick up the changes of the dashboards and automatically reload them.